# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fedcm/fedcm-reject-invalid-responses.https.html.ini
# Commit: da37e2d8a175
# Full Hash: da37e2d8a175fda08a98920d820dda0899b6aedf
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 15b92f98f62c -r da37e2d8a175 testing/web-platform/meta/fedcm/fedcm-reject-invalid-responses.https.html.ini
--- a/testing/web-platform/meta/fedcm/fedcm-reject-invalid-responses.https.html.ini	Thu Jun 12 18:16:52 2025 +0000
+++ b/testing/web-platform/meta/fedcm/fedcm-reject-invalid-responses.https.html.ini	Thu Jun 12 18:16:53 2025 +0000
@@ -1,20 +1,16 @@
 [fedcm-reject-invalid-responses.https.html]
-  expected: TIMEOUT
+  expected: [TIMEOUT, OK]
   [Reject when provider list is empty]
     expected: FAIL
 
   [Reject when configURL is missing]
-    expected: FAIL
+    expected: [FAIL, PASS]
 
   [Reject when configURL is invalid]
-    expected:
-      if os == "mac": [FAIL, TIMEOUT]
-      FAIL
+    expected: [FAIL, PASS, TIMEOUT, NOTRUN]
 
   [Reject when clientId is empty]
-    expected:
-      if os == "mac": [TIMEOUT, NOTRUN]
-      TIMEOUT
+    expected:[FAIL, PASS, TIMEOUT, NOTRUN]
 
   [Reject when clientId is missing]
-    expected: NOTRUN
+    expected: [FAIL, PASS, TIMEOUT, NOTRUN]